build/Targets/Settings.props

```diff
@@ -49,6 +49,11 @@
     <VisualStudioCodename>Dev$(VisualStudioReferenceMajorVersion)</VisualStudioCodename>
     <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
 
+    <!-- When building WPF / VSIX projects MSBuild will create a temporary project with an extension of
+         tmp_proj.  In that case the SDK is unable to determine the target language and cannot pick
+         the correct import.  Need to set it explicitly here. -->
+    <LanguageTargets Condition="'$(ProjectLanguage)' != ''">$(MSBuildToolsPath)\Microsoft.$(ProjectLanguage).targets</LanguageTargets>
```

Comment:
Technically compile items can be empty and extension doesn't necessarily mean language. Crazy people could put C# code into .vb files if they choose. This infrastructure was already in place hence just leveraging it for this change.